src/csfs.jl

```diff
@@ -1,3 +1,34 @@
+"""
+    struct CSF
+
+Represents a single _configuration state function_ (CSF) adapted to angular momentum
+symmetries. Depending on the type parameters, it can represent both non-relativistic CSFs in
+[``LS``-coupling](@ref) and relativistic CSFs in [``jj``-coupling](@ref).
+
+A CSF is defined by the following information:
+
+* The configuration, i.e. an ordered list of subshells together with their occupations.
+* A list of intermediate coupling terms (including the seniority quantum number to label
+  states in degenerate subspaces) for each subshell in the configuration.
+* A list of coupling terms for the coupling tree. The coupling is assume to be done by first
```

Comment:
the coupling is assume**d**
to be done by first **coupling**

Reply:
Fixed in a031847.